clients/user-services/jdbc-user-service/src/main/resources/db.properties

```diff
@@ -0,0 +1,3 @@
+db.url=jdbc:postgresql://172.23.0.2:5432/test
```

Comment:
is it a general properties file, which is placed not in tests directory? then why test database used here?)